README.md

```diff
@@ -6,6 +6,10 @@ For more information about the project, visit [http://projectelectron.rockarch.o
 
 Project updates are also available on [Bits & Bytes](http://blog.rockarch.org/), the RAC's blog.
 
+## bibliography
+
+The `/bibliography` directory contains an annotated bibliography created during the discovery phase of Project Electron.
```

Comment:
Maybe specify that it covers readings from topics related to data modeling, linked data, and archival data?

Reply:
Good call. Changed!